pandas/tests/io/formats/style/test_highlight.py

```diff
@@ -63,3 +63,65 @@ def test_highlight_minmax_ext(self, f, kwargs):
             df = self.df
         result = getattr(df.style, f)(**kwargs)._compute().ctx
         assert result == expected
+
+    @pytest.mark.parametrize(
+        "kwargs",
+        [
```

Comment:
I'm confused about how these all pass when there is self.randn(2) in self.df, which often will be outside of [0, 1]. Maybe I've missed something.

Reply:
the random seed is set so self.df is always: [[0, -0.6], [1, -1.2]].
where negatives are included in the potential highlighting range, the `subset` argument is used to restrict application only to col "A".

Reply:
incidentally I don't know why random is used. just complicates things, but is legacy from my perspective.

Reply:
Yes, I'm more used to other conventions around randomisation in testing. (As far as I'm concerned, if there is randomisation on the global generator, the test should be presumed invariant to the initial random state.)

Given that the random df is not explicitly shown near this test, could you please include a comment explaining, therefore, why these parameters make sense: even just show the df, or set it explic itly.

Reply:
i have set the df explicitly keeping the values same as previous. all tests still pass, of course.